doc/vhpidirect/examples/arrays.rst

```diff
@@ -0,0 +1,56 @@
+.. program:: ghdl
+.. _COSIM:VHPIDIRECT:Examples:arrays:
+
+Arrays
+######
+
+.. _COSIM:VHPIDIRECT:Examples:arrays:logicvectors:
+
+:cosimtree:`logicvectors <vhpidirect/arrays/logicvectors>`
```

Comment:
There's a mismatch here. In all other places, `logicvector` (singular) is used instead of `logicvectors` (plural). I'm good with either of them, but it needs to match :D